packages/apollo/src/utils/getOperationManifestFromProject.ts

```diff
@@ -27,54 +16,22 @@ export function getOperationManifestFromProject(
   const manifest = Object.values(
     project.mergedOperationsAndFragmentsForService
   ).map(operationAST => {
-    // While this could include dropping unused definitions, they are
-    // kept because the registered operations should mirror those in the
-    // client bundle minus any PII which lives within string literals.
-    const printed = printWithReducedWhitespace(
-      sortAST(hideCertainLiterals(operationAST))
-    );
+    const printed = defaultEngineReportingSignature(operationAST, "");
 
     return {
       signature: manifestOperationHash(printed),
       document: printed,
       metadata: {
-        engineSignature: engineSignature(operationAST)
+        engineSignature: printed
```

Comment:
fwiw I don't think anyone reads this field ever and (verify first) it could just be dropped?

Reply:
@glasser You're correct that we do not use this anywhere at the moment.  The original intention here was to provide us a way to match up operations which are registered with signatures in our trace warehouse, which are normalized by the `engineSignature`.    Do you think that association would be possible, and provide us a way to correlate this to operations in the Engine UI, if we maintained this signature?

Of course, if we do things based on operation name, this would become much easier.

Reply:
I would love us to move towards the Engine UI (a) mostly caring about operation name and (b) moving towards the same normalization used here (eg, perhaps it would still hide literals instead of banning them, but it would warn when it does so with something like "we hid the literal but we think you shouldn't use them and also it would mean you can't use operation registry")

Reply:
Removing this field altogether would violate the `RegisteredOperationInput` type, and I'd prefer to defer an update to the mutation and types until 2.0 work if possible.

See also: https://github.com/apollographql/apollo-tooling/pull/1027#discussion_r260871620

Reply:
You could pass the empty string or 'unused'?

Reply:
Resolved in 3e8da9035fc29f7b6a72ff155fd5a864e6aed9e4